docs/content/en/docs/getting-started/try-tetragon-linux.md

```diff
@@ -0,0 +1,406 @@
+Kubernetes), you can see the [container deployment](/docs/getting-started/deployment/container)
+and [package deployment](/docs/getting-started/deployment/package) guides for
+that. This is just a walkthrough to try and experiment Tetragon for the first
+time.
+{{< /caution >}}
+
+## Start Tetragon
+
+The easiest way to start experimenting with Tetragon is to run it via Docker
+using the released container images. If you prefer running directly the
+binaries without Docker, please refer to the
+[development setup](/docs/contribution-guide/development-setup) to see how to
+build the binaries.
+
+```shell
+docker run --name tetracontainer --rm                \
+    --pid=host --cgroupns=host --privileged          \
+    -v /sys/kernel/btf/vmlinux:/var/lib/tetragon/btf \
+    quay.io/cilium/tetragon:{{< latest-version >}} /usr/bin/tetragon
+```
+
+Let's break down the previous command:
+- `--name tetracontainer` names our container so that we can refer to it later
+  via a simple name.
+- `--rm` will make Docker delete the container once it exists.
+- `--pid=host` is needed to read the procfs for gathering context on processes
+  started before Tetragon.
+- `--cgroupns=host` is needed to detect container feature.
+- `--privileged` is needed for Tetragon to load its BPF programs.
+- `-v /sys/kernel/btf/vmlinux:/var/lib/tetragon/btf` mounts the BTF file inside
+  the Tetragon container filesystem. Tetragon needs a BTF file corresponding to
+  your kernel version to load its BPF programs.
+- `quay.io/cilium/tetragon:{{< latest-version >}}` is the latest release of
+  Tetragon available.
+
+## Observe Tetragon base events
+
+With this default configuration, Tetragon already loaded its base sensors to
+perform [process lifecycle observability](/docs/tetragon-events/process-lifecycle/).
+
+To quickly see the events, you can use the `tetra` CLI already shipped in the
+Tetragon container that was just started, it will connect to the Tetragon gRPC
+server listening on `localhost:54321`. In another terminal, type the following
+command:
+
+```shell
+docker exec tetracontainer tetra getevents -o compact
+```
+
+In a different terminal, you can start a shell like `sh` and start executing
+programs. Here, the commands executed in the shell where `ls`, `uname -a`,
+`whoami`, and `exit`, the output should be similar to this:
+
+```
+🚀 process  /usr/bin/sh
+🚀 process  /usr/bin/ls
+💥 exit     /usr/bin/ls  0
+🚀 process  /usr/bin/uname -a
+💥 exit     /usr/bin/uname -a 0
+🚀 process  /usr/bin/whoami
+💥 exit     /usr/bin/whoami  0
+💥 exit     /usr/bin/sh  0
+```
+
+You can see the start of `sh` first, then the execution of the multiple
+programs, `ls`, `uname` and `whoami`, and finally the exit of `sh`.
+
+{{< note >}}
+Since Tetragon is running directly on the host, you may observe a lot of
+unrelated from various process executions on your machine unrelated to what you
+typed during the shell session. Indeed, Tetragon is monitoring every process
```

Comment:
```suggestion
Since Tetragon monitors all processes on the host  host, you may observe events
unrelated to what was
typed in the shell session. Indeed, Tetragon is monitoring every process
```